allensdk/test/brain_observatory/behavior/behavior_project_cache_data_model/test_behavior_project_cache.py

```diff
@@ -6,6 +6,7 @@
 from allensdk.test_utilities.custom_comparators import safe_df_comparison
 
 
+@pytest.mark.skip()
```

Comment:
So what's up with skipping these tests? Are these tests that are no longer relevant in which case why don't we just remove them or are they just to make this ticket work in which case I worry about losing testing for these functions.

Reply:
These tests were written by others long ago and aren't testing the way we generate tables for vbo release. It doesn't seem worthwhile to spend the mental energy to update these tests.
I will delete these and write new tests

Reply:
Sounds good to me.